net/local/local_sockif.c

```diff
@@ -94,6 +100,10 @@ const struct sock_intf_s g_local_sockif =
   local_close,       /* si_close */
   local_ioctl,       /* si_ioctl */
   local_socketpair   /* si_socketpair */
+#ifdef CONFIG_NET_SOCKOPTS
+  local_getsockopt,  /* si_getsockopt */
+  local_setsockopt,  /* si_setsockopt */
```

Comment:
```suggestion
  , local_getsockopt /* si_getsockopt */
  , local_setsockopt /* si_setsockopt */
```
here and other similar places